### ISSUE TITLE
Tiles size

### ISSUE THREAD

**Issue by wil-m** [zbxwallboard/ZbxWallboard#7]
Hello and thank you for great module! It is not really issue, but more like request/question. Is it possible to display all items with fixed size of tile? even for scrolling price. My point: my superiors would like to have dashboard like yours, but with all items displayed (even without problems), but while any problem appears, it will appear on the top of the screen with red color. All other items remains green and below. If it is not goal of your module, is it possible to have fixed size of tile, please? I can see only one huge tile while having one problem or 99 tiny tiles while having 99 problems. thank you!

![Screenshot from 2020-09-26 21-43-06](https://user-images.githubusercontent.com/1379719/94349074-7de58480-0041-11eb-844a-9bf91e51a007.png)

**Comment by mjaepel** [comment]
You can remove tile scaling easy by removing line 75 in monitoring.wallboard.view.php.
I am on vacation currently. So it will take some time to get a deeper look inside 😉 Maybe next week.

**Comment by wil-m** [comment]
Thank you, enjoy your vacation!

**Comment by mjaepel** [comment]
I added an flag in filter options to disable the tile scaling. So no code changes are needed any more.
It's not compliant with MVC model but I used the original controller of problem view because the original zabbix code does all what we need here. It's not possible to add MVC-compliant custom flags there with the current Zabbix version. :(

Also checked the second whish to show also the triggers with OK state. But this also not possible with reused original controller of problem view. I have to duplicate the whole code or have to do some heavy non-MVC compliant workarounds. I won't do this at the moment.
Maybe Zabbix change the coding style in later versions then we could check it again. Or somebody will do it and create a merge request. ;)

**Comment by wil-m** [comment]
Thank you for your fast reply and work @Firewire2002 ! I understand that my superiors does have very unique request, which is not supported by Zabbix itself. Thanks a lot!

> I added an flag in filter options to disable the tile scaling. So no code changes are needed any more.
> It's not compliant with MVC model but I used the original controller of problem view because the original zabbix code does all what we need here. It's not possible to add MVC-compliant custom flags there with the current Zabbix version. :(
> 
> Also checked the second whish to show also the triggers with OK state. But this also not possible with reused original controller of problem view. I have to duplicate the whole code or have to do some heavy non-MVC compliant workarounds. I won't do this at the moment.
> Maybe Zabbix change the coding style in later versions then we could check it again. Or somebody will do it and create a merge request. ;)

**Comment by mjaepel** [comment]
That's not correct. There is a Dashboard/Screen widget "Triggeroverview" which does exactly what you want ... but as table.
My module has a reference on the code for problem view. So I can't change the datasource without duplicating the whole code and modify it.

This would be easier in the standalone version. Because there is the complete code from me. It's easy there to add an additional data source. 
Zabbix 5.0 is the first release with UI module support and they have several improvements for this on the roadmap. Maybe it will be easier in the next releases.
